.github/workflows/release.yml

```diff
@@ -95,7 +127,7 @@ jobs:
           echo "major=${patch%.*.*}" >> $GITHUB_OUTPUT
       - name: Build and Push release
         uses: docker/build-push-action@v6.7.0
-        if: github.event.release.prerelease == false
+        if: !github.event.workflow_dispatch && github.event.release.prerelease == false
```

Comment:
does this work in this syntax here ?

Reply:
I honestly don't know, and can't seem to find reliable info about it